temperature.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+
+float convertFahrToCelsius(const float fahrenheit);
+float convertCelsiusToKelvin(const float celsius);
+
+int main() {
+  float fahrenheit, celsius, kelvin;
+
+  printf("Fahrenheit: ");
+  scanf("%f", &fahrenheit);
+
+  celsius = convertFahrToCelsius(fahrenheit);
+  kelvin = convertCelsiusToKelvin(celsius);
+  printf("\nCelsius: %.2f\n", celsius);
+  printf("Kelvin: %.2f\n", kelvin);
```

Comment:
Consider just calling `convertCelsiusToKelvin` directly inside `printf` instead of declaring a `kelvin` variable. Like so.
```suggestion
  celsius = convertFahrToCelsius(fahrenheit);

  printf("\nCelsius: %.2f\n", celsius);
  printf("Kelvin: %.2f\n", convertCelsiusToKelvin(celsius));
```